drwmutex.go

```diff
@@ -379,61 +375,28 @@ func (dm *DRWMutex) ForceUnlock() {
 
 // sendRelease sends a release message to a node that previously granted a lock
 func sendRelease(c NetLocker, name, uid string, isReadLock bool) {
-
-	backOffArray := []time.Duration{
-		30 * time.Second, // 30 secs
-		1 * time.Minute,  // 1 min
-		3 * time.Minute,  // 3 min
-		10 * time.Minute, // 10 min
-		30 * time.Minute, // 30 min
-		1 * time.Hour,    // 1 hour
+	// All client methods issuing RPCs are thread-safe and goroutine-safe,
+	// i.e. it is safe to call them from multiple concurrently running goroutines.
```

Comment:
We can remove this comment as we are no longer doing this in a separate go routine.